src/test/mathutiltest.cpp

```diff
@@ -64,19 +64,23 @@ TEST_F(MathUtilTest, Denormal) {
     _MM_SET_DENORMALS_ZERO_MODE(_MM_DENORMALS_ZERO_OFF);
     _MM_SET_FLUSH_ZERO_MODE(_MM_FLUSH_ZERO_OFF);
 
-    volatile float fDenormal = std::numeric_limits<float>::min() / 2.0f;
+    volatile float fDenormal = std::numeric_limits<float>::min();
+    fDenormal /= 2.0f;
```

Comment:
Thank you! Please add a comment why the operations are written in this style including the `volatile` keyword.